packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -1271,3 +1280,183 @@ export function validateEventListenerTarget(
   }
   return false;
 }
+
+export const supportsTestSelectors = true;
+
+export function findRootFiber(node: Instance): null | Fiber {
+  const stack = [node];
+  while (stack.length > 0) {
+    const current = stack.shift();
+    if (isContainerMarkedAsRoot(current)) {
+      const root = ((getInstanceFromNodeDOMTree(current): any): Fiber);
+      return root.stateNode.current;
```

Comment:
We should try not to let the Fiber type leak into the host config. We already do in other places, unfortunately. Could this return the Container type (`root.stateNode`) instead?

Reply:
Yeah still thinking through the right API for the host config methods (state node vs fiber). Easy to change :)